bin/utils/brk_up_large_scn_host_configs

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+#                         License
+#
+# Copyright (C) 2024  David Valin dvalin@redhat.com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+#
+# This script takes a Zathras scenario file that has large host lines in it and breaks
+# it down to be about 1024 characters without breaking the expected Zathras format.
+#
+gl_scenario_to_run=""
+
+exit_out()
+{
+	echo $1
+	exit $2
+}
+
```

Comment:
This is fine for now, but `exit_out` will probably be used across many files, so it should be in a library shell file _somewhere_ that the scripts will `source` from.